server.rb

```diff
@@ -0,0 +1,189 @@
+require "sinatra/base"
+require "sqlite3"
+require "nokogiri"
+require 'active_support'
+require 'json'
+require 'securerandom'
+
+class Server < Sinatra::Base
+
+  # {
+  #  to: <UUID>
+  #  from: <UUID>
+  #  message: <String>
+  # }
+
+  post '/messages/:format?' do
```

Comment:
Both of the `post` and `get` actions don't have any authentication/authorisation steps. I guess it is alright to keep things simple but for security we may want to consider adding this in the future.